frappe/installer.py

```diff
@@ -266,7 +266,7 @@ def install_app(name, verbose=False, set_as_patched=True):
 			return
 
 	if name != "frappe":
-		add_module_defs(name)
+		add_module_defs(name, ignore_if_duplicate=True)
```

Comment:
```suggestion
		add_module_defs(name, ignore_if_duplicate=force)
```

need to add the same in the function definition 
https://github.com/frappe/frappe/blob/da32a2615e710b5c450ccf3b7dece160ee76763e/frappe/installer.py#L576

as well as 👇 
https://github.com/frappe/frappe/blob/da32a2615e710b5c450ccf3b7dece160ee76763e/frappe/installer.py#L582